src/Core/Resolvers/QueryExecutor.cs

```diff
@@ -805,5 +832,21 @@ private void ValidateSize(long availableSizeBytes, long sizeToBeReadBytes)
                     subStatusCode: DataApiBuilderException.SubStatusCodes.ErrorProcessingData);
             }
         }
+
+        private void AddDbExecutionTimeToMiddlewareContext(long time)
```

Comment:
per pr description on testing: 
> Since this is only a timer addition and no logical change. existing unit tests should cover code.

You're adding key/value pair to the httpcontext. Wouldn't it be good to test to ensure that this value exists and is added as expected? there is no existing test coverage to check that the contents were updated with this new content.